pkg/cmd/install.go

```diff
@@ -54,12 +54,15 @@ func installCmd(rootFlags *pflag.FlagSet) *cobra.Command {
 		Use:   "install",
 		Short: "Install Kubernetes",
```

Comment:
Nit: should we add `[DEPRECATED]` here as well? We have it for `upgrade`.

Reply:
Changes